File: ImplementationEnC/Exercice1_1.c
#include <stdlib.h>
#include <stdio.h>

int main(){
	char sexe[5];
	int age;
	char gen1[]="homme";
	char gen2[]="femme";
	printf("le sexe : ");
	scanf("%s" , &sexe);
	printf("l'age : ");
	scanf("%d" , &age);
	if( strcmp(gen1,sexe) && age >= 20 ){
		printf("vous devez payer limpot");
    }
	else if ( strcmp(gen2,sexe) && age >= 18 && age <= 35){
		printf("vous devez payer limpot");
	}
	else{
		printf("ne payee pas limpot");
	}


}

File: ImplementationEnPython/Exercice1_1.py
bool = True
while bool:
    sexe = input("le Sexe : ")
    lage = int(input("l'age : "))
    if sexe == "homme" and lage >= 20 :
        print("vous devez payer l'impot ")
    elif sexe == "femme" and lage >= 18 and lage <= 35 :
        print("vous devez payer l'mpot")
    else :
        print("ne payee pas l'impot")
    
